packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/UnbrandedTypeSpecClientOptions.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace UnbrandedTypeSpec
 {
-    /// <summary></summary>
+    /// <summary> Client options for UnbrandedTypeSpecClient. </summary>
     public partial class UnbrandedTypeSpecClientOptions : ClientPipelineOptions
```

Comment:
Do we not expect this to be filled out with service versions?

Reply:
this test project doesn't specify any service versions in the spec.

Reply:
Is it worth adding to the spec?

Reply:
I can, but I avoided adding it to the spec because @m-nash wants us to move toward a model where we rely more on unit tests vs generated code. @m-nash  - any concerns if I add versions to the spec for this project?